libs/ui/src/ui_strings/app_strings.tsx

```diff
@@ -59,6 +110,16 @@ export const appStrings = {
     </UiString>
   ),
 
+  labelNumVotesRemaining: () => (
+    <UiString uiStringKey="labelNumVotesRemaining">
+      Votes remaining in this contest:
+    </UiString>
+  ),
+
+  labelWriteInParenthesized: () => (
+    <UiString uiStringKey="labelWriteInParenthesized">(write-in)</UiString>
+  ),
+
   number: (value: number) => <NumberString value={value} />,
 
   numSeatsInstructions: (numSeats: number) =>
```

Comment:
nit: I see that you've started converging on the pattern of prefixing keys, e.g. `label...`, `instructions...`, `button...`, `title...`, which I like! This one could be renamed `instructionsNumSeats` accordingly

Reply:
yeah, good call! will update the older keys to reflect the new pattern

Reply:
✅  [7b302e1](https://github.com/votingworks/vxsuite/pull/4143/commits/7b302e1969c504b43d28b1655936145b6ab3d998)

will be following up with a change to remove the `numVotes*`/`numSeats*` strings altogether, in favour of non-interpolated ones